codes/c/chapter_backtracking/preorder_traversal_iii_template.c

```diff
@@ -6,96 +6,84 @@
 
 #include "../utils/common.h"
 
+// 假设结果长度不超过 100
+#define MAX_SIZE 100
+
+TreeNode *path[MAX_SIZE];
+TreeNode *res[MAX_SIZE][MAX_SIZE];
+int pathSize = 0, resSize = 0;
+
```

Comment:
Same comments as in the previous document(`codes/c/chapter_backtracking/preorder_traversal_ii_compact.c`)